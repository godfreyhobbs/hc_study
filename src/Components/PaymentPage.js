import React, { Component } from 'react';
import Header from './Header';
import '../App.css';

class PaymentPage extends Component {


  handleClick() {
    this.props.callback("GrantAccess");
  }

    render() {
    return (
      <div>
        <div>
          <h1>Promise of Payment</h1>
          <p>You will be paid to account {this.props.account} within 2 business days.</p>

          <p>Study Number: 12345</p>
          <p>Participant ID: 123</p>
        </div>
      </div>
    );
  }
}

export default PaymentPage;
