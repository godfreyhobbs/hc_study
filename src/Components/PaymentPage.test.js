import React from 'react';
import ReactDOM from 'react-dom';
import PaymentPage from './PaymentPage';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<PaymentPage />, div);
  ReactDOM.unmountComponentAtNode(div);
});



it('renders correctly', () => {
  const tree = renderer
.create(<PaymentPage />)
   .toJSON();
expect(tree).toMatchSnapshot();
});
