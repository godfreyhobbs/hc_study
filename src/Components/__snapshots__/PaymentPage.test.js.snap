// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <h1>
      Promise of Payment
    </h1>
    <p>
      You will be paid to account 
       within 2 business days.
    </p>
    <p>
      Study Number: 12345
    </p>
    <p>
      Participant ID: 123
    </p>
  </div>
</div>
`;
